docs/pages/getting-started.md

```diff
@@ -30,4 +30,50 @@ npm i fundamental-styles --save
 
 ```
 https://unpkg.com/fundamental-styles@{{site.library_version}}/dist/fundamental-styles.css
-```
\ No newline at end of file
+```
+
+## Fonts
+
+
+SAP 72 font family was designed from the ground up and made it possible to meet SAP Fiori’s typographic requirements, including - legibility enhancements, font styles, brand voice, character set and language support.
+
+<br>
+<p class="docs-type_SAP">
+<span class="docs-type_drop-cap">72</span>
+A B C D E F G H I J K L M N O P Q R S T U V W X Y Z <br>
+a b c d e f g h i j k l m n o p q r s t u v w x y z <br>
+1 2 3 4 5 6 7 8 9 0 <br>
+​‘ ? ’ “ ! ” ( % ) [ # ] { @ } / &amp; \ &lt; - + ÷ × = &gt; ® © $ € £ ¥ ¢ : ; , . *
+</p>
+<br>
+<br>
+<a href="https://experience.sap.com/72/" target="_blank">Learn More about the SAP 72 Font Family</a>
+<br>
+
+
+## Icons
+
+See the [Icon Component]({{site.baseurl}}/components/icon.html) for a list of icon class names. See [Project Configuration](#Project-configuration) below for instructions to include SAP Fiori 3 icons in your project.
+
```

Comment:
I didn't really have a lot to say about icons - is there a public website about fiori 3 icons like there is for fonts?